docs/advanced/payments/existing-domain-events.mdx

```diff
@@ -0,0 +1,167 @@
+---
+sidebar_position: 5.3
+title: Existing Domain Events
+description: See the detailed list of existing Domain Events.
+---
+
+# Early Notification Events
```

Comment:
**issue:** these aren't domain events. It is domain value objects or  commands.\
Mentioning it here makes it more complex to understand

**suggestion:** remove this section from here, and document it in a separate page or a section about implementing notification processors

Reply:
move to a new NotificationProcessor section